File: app/src/main/java/com/elixsr/portforwarder/ui/preferences/AdvancedSettingsActivity.java
package com.elixsr.portforwarder.ui.preferences;

import android.os.Bundle;
import androidx.appcompat.widget.Toolbar;
import android.view.View;

import com.elixsr.portforwarder.R;
import com.elixsr.portforwarder.ui.BaseActivity;

public class AdvancedSettingsActivity extends BaseActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_advanced_settings);
        Toolbar toolbar = getActionBarToolbar();
        setSupportActionBar(toolbar);

        toolbar.setNavigationIcon(R.drawable.ic_arrow_back_24dp);
        toolbar.setNavigationOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                onBackPressed();
            }
        });

        getFragmentManager().beginTransaction().replace(R.id.advanced_settings_container, new AdvancedSettingsFragment()).commit();
    }

}

File: app/src/main/java/com/elixsr/portforwarder/ui/preferences/AdvancedSettingsActivity.kt
package com.elixsr.portforwarder.ui.preferences

import android.os.Bundle
import android.view.View
import com.elixsr.portforwarder.R
import com.elixsr.portforwarder.ui.BaseActivity

class AdvancedSettingsActivity : BaseActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_advanced_settings)
        val toolbar = actionBarToolbar
        setSupportActionBar(toolbar)
        toolbar!!.setNavigationIcon(R.drawable.ic_arrow_back_24dp)
        toolbar.setNavigationOnClickListener { v: View? -> onBackPressed() }
        fragmentManager.beginTransaction().replace(R.id.advanced_settings_container, AdvancedSettingsFragment()).commit()
    }
}